create-security-certificates.md

```diff
@@ -2,3 +2,68 @@
 title: Create Security Certificates
 toc: false
 ---
+
+A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` command with the appropriate subcommand and flags. 
+
+{{site.data.alerts.callout_info}}When you create node and client certificates, you need access to a local copy of the CA certificate and key. Therefore, it's recommended to create all certificates and keys in one place and then distribute node and client certificates and keys appropriately and store the CA key in a safe place. For a walkthrough of this process, see <a href="https://www.cockroachlabs.com/docs/manual-deployment.html">Manual Deployment</a>.{{site.data.alerts.end}}
```

Comment:
Generating all your keys in one place is not a best practice when it comes to security (see e.g. http://security.stackexchange.com/questions/102390/what-is-autocsr-and-is-it-secure/102408). However, we haven't implemented anything better with the `cockroach cert` tool yet (the right thing to do is to generate a key and CSR on the target node, transfer the CSR to the machine with the CA key, and get a cert that you send to the target node). I would therefore qualify the "it's recommended to" with "when using `cockroach cert`, it's recommended to".

We also need another bigger warning about the CA key: It is not recommended to copy the CA key to every node, but if you lose it you will not be able to add new nodes to your cluster. You must therefore store it somewhere safe and keep a backup.